src/archanaNJun20/DuplicateNumber.java

```diff
@@ -0,0 +1,17 @@
+package archanaNJun20;
+
+public class DuplicateNumber {
+	void printDuplicateNumber(int array1[]){
+		for(int i=0;i<array1.length;i++) {
+			for(int j=i+1;j<array1.length;j++){
```

Comment:
Better to use index and innerIndex instead of i and j.

Reply:
The code is updated based on comments.